src/content/learn/manipulating-the-dom-with-refs.md

```diff
@@ -376,40 +376,31 @@ export default function MyForm() {
 
 </Sandpack>
 
-To help you notice the issue, React also prints an error to the console:
-
-<ConsoleBlock level="error">
```

Comment:
and here instead you'll describe the crash

Reply:
I just added a brief explanation of what causes the crash, because trying the example will show the console output. If you'd like, I can add back the `<ConsoleBlock />` that shows the console output anyway:

```
Cannot read properties of null (reading 'focus')
```

Reply:
yup let's add that